openml/runs/functions.py

```diff
@@ -1094,6 +1094,19 @@ def list_runs(offset=None, size=None, id=None, task=None, setup=None,
         List of found runs.
     """
 
+    if id is not None and (not type(id) is list):
```

Comment:
When checking for types in Python, the best practice is to use the `isinstance(object, type)` method, e.g. `isinstance(id, list)`, this is because [isinstance also allows for the object to be a subclass](https://premaseem.wordpress.com/2017/10/11/python-difference-between-type-and-isinstance/).